notebooks/quickstart_tutorial.ipynb

```diff
@@ -232,7 +232,7 @@
    "outputs": [],
    "source": [
     "# Select the annotation model\n",
-    "cas_model_name = '<model_name>'"
+    "cas_model_name = 'cellarium-pca-default-v1'"
```

Comment:
I get why you're doing this but keep in mind it's gonna be a pain to maintain this (like if you change the default model at some point and these notebooks are in the wild).  Might make sense to make the APIs take a flag that states to use the default model (or maybe if not model is specified, to just use the default)

Reply:
Done.